src/terminal/adapter/DispatchTypes.hpp

```diff
@@ -261,19 +261,19 @@ namespace Microsoft::Console::VirtualTerminal::DispatchTypes
     enum GraphicsOptions : size_t
     {
         Off = 0,
-        BoldBright = 1,
+        Intense = 1,
         // The 2 and 5 entries here are for BOTH the extended graphics options,
         // as well as the Faint/Blink options.
         RGBColorOrFaint = 2, // 2 is also Faint, decreased intensity (ISO 6429).
         Italics = 3,
         Underline = 4,
-        BlinkOrXterm256Index = 5, // 5 is also Blink (appears as Bold).
+        BlinkOrXterm256Index = 5, // 5 is also Blink.
         RapidBlink = 6,
         Negative = 7,
         Invisible = 8,
         CrossedOut = 9,
         DoublyUnderlined = 21,
-        NotBoldOrFaint = 22,
+        NotIntenseOrFaint = 22,

```

Comment:
My brain really wanted this to be `NotIntenseNorFaint`... but I can live with it if you don't want to change it. :P (Nor is such an underused conjunction that I love to give it a fighting chance...)

Reply:
in that case it should be Neither..Nor right?

Reply:
Oh sure. `NeitherIntenseNorFaint`. Whatever. I just like NOR ok?!

Reply:
I don't feel that strongly about it, but I'd kind of prefer it if the disabling attributes were consistently using a `Not` prefix (although the `NoUnderline` attribute already breaks that rule). I'll let you guys decide though. Just give me an explicit command and I'll make the change.

Reply:
I'm fine with it as it is in this PR :)

Reply:
Yeah I'm honestly fine with it. I'm just a goof. Thanks, @j4james.